gneiss/util.py

```diff
@@ -0,0 +1,117 @@
+def match(x, y, intersect=False):
+    """ Sorts samples in metadata and contingency table in the same order.
+
+    Parameters
+    ----------
+    x : pd.DataFrame
```

Comment:
I noticed that `x` and `y` are both DataFrames but conceptually are two different things. I would recommend to rename the variable to `cont_table` and `metadata` or similar, but at least more meaningful that `x` and `y`